index.html

```diff
@@ -0,0 +1,145 @@
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>WOS Concert</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+<body>
+    <div class="wrapper">
+        <header class="head-navbar">
+            <div class="mobile-nav">
+                <nav class="with-hamburger">
+                    <div class="slidediv">
+                        <ul class="olist">
+                            <a class="olist-a" href='about.html'><li>About</li></a>
+                            <a class="olist-a" href="#main-program"><li>Program</li></a>
+                            <a class="olist-a" href="#join-event-button"><li>Join</li></a>
+                            <a class="olist-a" href="#partners"><li>Sponsor</li></a>
+                            <a class="olist-a" href="#"><li>News</li></a>
+                        </ul>
+                    </div>
+                    <button class="menu-button">
+                        <div class="ham">
+                            <div class="bar1"></div>
+                            <div class="bar2"></div>
+                            <div class="bar3"></div>
+                        </div>
+                    </button>
+                </nav>
+            </div>
+            <div class="desktop-nav">
+                <div class="header-bar">
+                    <ul class="header-bar-list">
+                        <li><ion-icon name="logo-facebook"></ion-icon></li>
+                        <li><ion-icon name="logo-twitter"></ion-icon></li>
+                        <li>English</li>
+                        <li>My Page</li>
+                        <li>Logout</li>
+                    </ul>
+                </div>
+                <nav class="with-logo-list">
+                    <div class="logo">
+                        <a class='logo-img-link' href='index.html'><img class="logo-img" src="./media/logo/WOSresize.png" alt="WOS Logo"></a>
+                    </div>
+                    <div class="nav-list">
+                        <ul class="nav-list-items">
+                            <li><a href='about.html'>About</a></li>
+                            <li><a href='#main-program'>Program</a></li>
+                            <li><a href='#join-event-button'>Join</a></li>
+                            <li><a href='#partners'>Sponser</a></li>
+                            <li><a href=''>News</a></li>
+                        </ul>
+                    </div>
+                    <div class="tour-button">
+                        <button class="wos-tour-button">WOS Tour</button>
+                    </div>
+                </nav>
+            </div>
+        </header>
+        <main class="maincontent">
+            <section class="head-section">
+                <div class="headings">
+                    <h2 class="headline-h2">"Hello! Sharing world"</h2>
+                <h1 class="headline-h1">White Orange Sky Summer Concert 2023</h1>
```

Comment:
**OPTIONAL** - You have done well so far, but to help improve the semantics of your page, it is advisable to let your `hx` tag follow an ascending hierarchy, i.e use an `h2`tag after an `h1` tag, use an `h3` tag after an `h2` tag e.t.c. Please fix this 🙏